src/commctl/cli.py

```diff
@@ -806,6 +873,47 @@ def add_cluster_commands(argument_parser):
     verb_parser.add_argument('name', help='Name of the cluster')
 
 
+def add_container_manager_commands(argument_parser):
+    """
+    Augments the argument parser with "container_manager" subcommands.
+
+    :param argument_parser: The argument parser to augment
+    :type argument_parser: argparser.ArgumentParser
+    """
+    _configure_parser(argument_parser, 'dispatch_container_manager_command')
+
+    # Note, commands follow a "subject-verb" or "subject-object-verb"
+    # pattern.  e.g. "host create" or "cluster upgrade start"
+
+    subject_subparser = argument_parser.add_subparsers(dest='command')
+
+    # Sub-command: cluster create
+    verb_parser = subject_subparser.add_parser('create')
+    verb_parser.required = True
+    verb_parser.add_argument(
+        '-t', '--type', help='Type of the container manager',
+        choices=('openshift', ), default='openshift')
```

Comment:
Not a blocker, but maybe hold off on this option until there's multiple choices.
"Type" isn't really an option if it can be anything you want as long as it's `"openshift"`.